CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### Fixed
 
+- Adjust payment terms on cost center
+
+## [1.11.2] - 2024-06-13
```

Comment:
you don't need to add this line, it will be added automatically after merged, but I do believe you need to rebase your PR (which will mostly like cause a conflict here)